### PR TITLE
docs: Improve Docs

### PR DESCRIPTION
**Pull request by Gustl22**
# Description

Improve the documentation for:

- Getting started: 
  - Fix typos from #1678
  - Call stop in low latency, missing completion event (#1489)
  - Set prefix for AssetSource via AudioCache (#1620, #1639)
- Troubleshooting guide:
  - Verify audio format, e.g. with Aconvert
  - Change heading size of web-cors-policy
- Example:
  - README: How to setup example with `melos`, tab descriptions
  - example.md: Add Simple (one-page) Example for pub.dev, see https://dart.dev/tools/pub/package-layout#examples (#1616)
  - Showcase for using play in `addPostFrameCallback`

See #1650 

## Checklist

- [x] The title of my PR starts with a [Conventional Commit] prefix (`fix:`, `feat:`, `refactor:`,
      `docs:`, `chore:`, `test:`, `ci:` etc).
- [x] I have read the [Contributor Guide] and followed the process outlined for submitting PRs.
- [ ] I have updated/added tests for ALL new/updated/fixed functionality.
- [x] I have updated/added relevant documentation and added dartdoc comments with `///`, where necessary.
- [ ] I have updated/added relevant examples in [example].

## Breaking Change

- [ ] Yes, this is a breaking change.
- [x] No, this is *not* a breaking change.

## Related Issues

#1650

[issue database]: https://github.com/bluefireteam/audioplayers/issues
[Contributor Guide]: https://github.com/bluefireteam/audioplayers/blob/main/contributing.md#feature-requests--prs
[Conventional Commit]: https://conventionalcommits.org
[example]: https://github.com/bluefireteam/audioplayers/tree/main/packages/audioplayers/example

### CODE REVIEW

**Comment by Gustl22** [comment]
If need to split in multiple Merge Requests, let me know ;D

File: packages/audioplayers/example/example.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
+Below example demonstrates a simple player app.
```
Comment (spydon) [inline packages/audioplayers/example/example.md:1]:
```suggestion
# Simple audio player app example
```

File: packages/audioplayers/example/lib/components/player_widget.dart [hunk 2]
```diff
@@ -3,6 +3,8 @@ import 'dart:async';
 import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/material.dart';
 
+// This code is also used in the example.md. Please keep it up to date.
+
```
Comment (spydon) [inline packages/audioplayers/example/lib/components/player_widget.dart:7]:
```suggestion
```

**Review by spydon — APPROVED** [review]
Very nice! 🍻

**Comment by Gustl22** [comment]
Also I'm not that fluent in english. So I don't mind if you have some bigger proposals.

**Comment by spydon** [comment]
> Thank you for reviewing. Also I'm not that fluent in english. So I don't mind if you have some bigger proposals.

I think it looks good. :)

File: packages/audioplayers/example/lib/components/player_widget.dart (hunk 2 above)
Comment (Gustl22) [reply packages/audioplayers/example/lib/components/player_widget.dart:7]: I wonder if we remove this line, one may thinks, it is only meant for the class `PlayerWidget` but it's also meant for the `_PlayerWidgetState`. But I think that's obvious.

File: getting_started.md [hunk 3]
```diff
@@ -322,6 +329,32 @@ It works as a cache because it keeps track of the copied files so that you can r
...
+await player.play(AssetSource('assets/audio/my-audio.wav'));
+```
+
+Set a different prefix for only one player (e.g. when using assets from another package):
+```dart
+final player = AudioPlayer();
+player.audioCache = AudioCache(prefix: 'packages/OTHER_PACKAGE/assets/')
+await player.play(AssetSource('other-package-audio.wav'));
+```
+
```
Comment (Gustl22) [inline getting_started.md:356]:
```suggestion
```

**Pull request merged**